src/Promitor.Integrations.Sinks.Statsd/StatsdMetricSink.cs

```diff
@@ -53,5 +69,24 @@ public Task ReportMetricAsync(string metricName, string metricDescription, doubl
 
             return Task.CompletedTask;
         }
+
+        public Task ReportCustomFormattedMetricAsync(string metricName, string metricDescription, double metricValue, Dictionary<string, string> labels)
+        {
+            Guard.NotNullOrEmpty(metricName, nameof(metricName));
+            Guard.NotNull(_statsDConfiguration.CurrentValue, nameof(_statsDConfiguration.CurrentValue));            
+
+            var bucket = JsonConvert.SerializeObject(new { 
+                Account = _statsDConfiguration.CurrentValue.Account, 
+                Namespace = _statsDConfiguration.CurrentValue.Namespace, 
+                Metric = metricName, 
+                Dims = labels 
+            });
+
+            _statsDPublisher.Gauge(metricValue, bucket);
+
+            _logger.LogTrace("Metric {MetricName} with value {MetricValue} was written to StatsD server", metricName, metricValue);
```

Comment:
This is same as default, let's consolidate
```suggestion
            LogMetricWritten(metricName, metricValue);
```